### PR TITLE
Migrate to Google Analytics 4

### PR DESCRIPTION
**Pull request by michaelglenister**
Issue: https://trello.com/c/T4EBvjA1/612-google-analytics-4

I added environment variables `GOOGLE_GA4_SCORECARD_ID` and `GOOGLE_GA4_DATA_ID` to the review app that enable GA4

These variables are already set in production in preparation to merge

### CODE REVIEW

File: assets/js/components/indicators.js [hunk 1]
```diff
@@ -263,12 +263,18 @@ export class IndicatorSection {
...
+      gtag('event', 'chart_averages', {
+        chart: `${this.key} provincial`,
+      });
     }));
 
     const $nationalButton = $(' <button class="button" style="display: unset">nationally</button>');
     $nationalButton.on('click', (() => {
       this.chart.loadMedians(this.formatMedians().national);
       ga('send', 'event', 'chart-averages', `${this.key} national`);
+      gtag('event', 'chart_averages', {
```
Comment (paulmwatson) [inline assets/js/components/indicators.js:275]: @michaelglenister is there a reason the `ga` is `national` but the `gtag` is `provincial`?

File: assets/js/webflow/import.js [hunk 2]
```diff
@@ -72,6 +72,14 @@ ga('create', '{{ GOOGLE_ANALYTICS_ID }}', 'auto');\
 {% endif %}\
 ga('send', 'pageview');\
 ");
+  addScriptToBody(window, {
+    src: 'https://www.googletagmanager.com/gtag/js?id={{ GOOGLE_GA4_TAG }}',
```
Comment (paulmwatson) [inline assets/js/webflow/import.js:76]: @michaelglenister is it right that we're adding Google Tag Manager but the variable is called `GOOGLE_GA4_TAG`? In other projects we use `TAG_MANAGER_CONTAINER_ID` for Google Tag Manager.

Comment (michaelglenister) [reply assets/js/webflow/import.js]: My thought process was I wanted a prefix that placed the variables next to each other in dokku config. I think GOOGLE_TAG_MANAGER_ID is better unless we really want to standardize env variable names

File: assets/js/components/indicators.js (hunk 1 above)
Comment (michaelglenister) [reply assets/js/components/indicators.js:275]: Copy pasta error, fixed

File: assets/js/components/indicators.js [hunk 3]
```diff
@@ -296,6 +302,9 @@ export class IndicatorSection {
       button.click(() => {
         this.chart.highlightCol(comparison.municipality.code);
         ga('send', 'event', 'chart-compare-highlight', `${this.key} ${comparison.municipality.code}`);
+        gtag('event', 'chart_compare_highlight', {
+          chart: `${this.key} ${comparison.municipality.code}`,
```
Comment (jbothma) [inline assets/js/components/indicators.js:306]: let's put this in `label` instead of `action`

File: assets/js/components/indicators.js [hunk 4]
```diff
@@ -263,12 +263,18 @@ export class IndicatorSection {
...
+        chart: `${this.key} provincial`,
+      });
     }));
 
     const $nationalButton = $(' <button class="button" style="display: unset">nationally</button>');
     $nationalButton.on('click', (() => {
       this.chart.loadMedians(this.formatMedians().national);
       ga('send', 'event', 'chart-averages', `${this.key} national`);
+      gtag('event', 'chart_averages', {
+        chart: `${this.key} national`,
```
Comment (jbothma) [inline assets/js/components/indicators.js:276]: let's put this in `label` instead of `action`

File: assets/js/components/indicators.js [hunk 5]
```diff
@@ -263,12 +263,18 @@ export class IndicatorSection {
     $provinceButton.on('click', (() => {
       this.chart.loadMedians(this.formatMedians().provincial);
       ga('send', 'event', 'chart-averages', `${this.key} provincial`);
+      gtag('event', 'chart_averages', {
+        chart: `${this.key} provincial`,
```
Comment (jbothma) [inline assets/js/components/indicators.js:267]: let's put this in `label` instead of `action`

File: assets/js/components/comparison-menu.js [hunk 6]
```diff
@@ -24,6 +24,10 @@ export default class ComparisonMenu {
       this.$dropdown.triggerHandler('w-close.w-dropdown');
 
       ga('send', 'event', 'compare-in-chart', `${key} ${option}`);
+      gtag('event', 'compare_in_chart', {
+        chart: `${key}`,
+        option: `${option}`,
```
Comment (jbothma) [inline assets/js/components/comparison-menu.js:29]: action

File: assets/js/components/comparison-menu.js [hunk 7]
```diff
@@ -24,6 +24,10 @@ export default class ComparisonMenu {
       this.$dropdown.triggerHandler('w-close.w-dropdown');
 
       ga('send', 'event', 'compare-in-chart', `${key} ${option}`);
+      gtag('event', 'compare_in_chart', {
+        chart: `${key}`,
```
Comment (jbothma) [inline assets/js/components/comparison-menu.js:28]: label

File: assets/js/pages/profile.js [hunk 8]
```diff
@@ -114,6 +114,9 @@ export default class ProfilePage {
     // track outbound links
     $('a[href^=http]').on('click', (e) => {
       ga('send', 'event', 'outbound-click', e.target.href);
+      gtag('event', 'outbound_click', {
+        target: e.target.href,
```
Comment (jbothma) [inline assets/js/pages/profile.js:118]: let's put this in `label` instead of `action`

File: assets/js/components/indicators.js (hunk 3 above)
Comment (michaelglenister) [reply assets/js/components/indicators.js]: Changed to label

File: assets/js/components/indicators.js (hunk 4 above)
Comment (michaelglenister) [reply assets/js/components/indicators.js]: Changed to label

File: assets/js/components/indicators.js (hunk 5 above)
Comment (michaelglenister) [reply assets/js/components/indicators.js]: Changed to label

File: assets/js/components/comparison-menu.js (hunk 6 above)
Comment (michaelglenister) [reply assets/js/components/comparison-menu.js]: Combined into label

File: assets/js/pages/profile.js (hunk 8 above)
Comment (michaelglenister) [reply assets/js/pages/profile.js]: Changed to label

**Review by jbothma — APPROVED** [review]

**Pull request merged**